site/en/docs/extensions/whatsnew/index.md

```diff
@@ -18,56 +18,57 @@ and related policy or other changes.
 
 ### Chrome 100: `i18n.getMessage()` support in extension service workers {: #m100-i18n-getmessage }
 
+January 27, 2022
+
 The [`chrome.i18n.getMessage()`](/docs/extensions/reference/i18n/#method-getMessage) API is now
-supported in extension service worker contexts. This change landed in Canary on January 27, 2022
-and is expected to reach Stable in Chrome 100.
+supported in extension service worker contexts.
+
+### Chrome 99: `match_origin_as_fallback` in Canary {: #canary-match-origin-as-fallback }
```

Comment:
```suggestion
### Chrome 99: match_origin_as_fallback in Canary {: #canary-match-origin-as-fallback }
```

Reply:
We don't do this in headings. The reasons for setting off code are not as critical in headings. It also tends to make documents look like ransom notes from old movies.

Reply:
Please check this throughout.

Reply:
Roger, will do.